src/models/propulsion/FGBrushLessDCMotor.h

```diff
@@ -0,0 +1,111 @@
+  with this program; if not, write to the Free Software Foundation, Inc., 59
+  Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+
+  Further information about the GNU Lesser General Public License can also be
+  found on the world wide web at http://www.gnu.org
+
+
+HISTORY
+--------------------------------------------------------------------------------
+1/01/2022   Created
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+SENTRY
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%*/
+
+#ifndef FGBrushLessDCMotor_H
+#define FGBrushLessDCMotor_H
+
+/*%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+INCLUDES
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%*/
+
+#include "FGEngine.h"
+
+/*%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+FORWARD DECLARATIONS
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%*/
+
+namespace JSBSim {
+
+/*%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+CLASS DOCUMENTATION
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%*/
+
+/** Models an electric brushless DC motor or more appropriately permanent magnet
+    synchronous motor.
+    FGElectric models an electric motor based on the configuration file
+    \<power> parameter.  The throttle controls motor output linearly from
+    zero to \<power>.  This power value (converted internally to horsepower)
+    is then used by FGPropeller to apply torque to the propeller.  At present
+    there is no battery model available, so this motor does not consume any
+    energy.  There is no internal friction.
+    @author David Culp
+  */
+
+/*%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+CLASS DECLARATION
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%*/
+
+class FGBrushLessDCMotor : public FGEngine
+{
+public:
+  /// Constructor
+  FGBrushLessDCMotor(FGFDMExec* exec, Element *el, int engine_number, FGEngine::Inputs& input);
+  /// Destructor
+  ~FGBrushLessDCMotor();
+
+  void Calculate(void);
+  double GetPowerAvailable(void) {return (HP * hptoftlbssec);}
+  double GetCurrentRequired(void) {return CurrentRequired;}
+  double getRPM(void) {return RPM;}
+  double CalcFuelNeed(void);
+  std::string GetEngineLabels(const std::string& delimiter);
+  std::string GetEngineValues(const std::string& delimiter);
+
+private:
+
+  // constants
+  double hptowatts;
+  double NoLoadCurrent;      // 0 torque current [A]
+  double CoilResistance;     // internal resistance [Ohm]
+  double PowerWatts;         // maximum engine power
+  double MaxCurrent;         // maximum current [A]
+  double MaxVolts;           // max voltage available from battery [V]
+  double VelocityConstant;   //.speed constant of brusless DC motors [RPM/V]
+  double TorqueConstant=6.1217;
```

Comment:
What is `TorqueConstant`?

Reply:
```suggestion
  constexpr double TorqueConstant = 6.1217;
```
If it's kept as is then it should be defined as `constexpr`.